tests/scikit_learn/etl/test_split.py

```diff
@@ -83,6 +83,70 @@ def test_split_randomness():
                                    'petallength', 'petalwidth'], size=slice_size))
 
 
+def test_split_fail_by_slice_size():
+    slice_size = 1  # No need to split in this case... is this value allowed in the GUI?
+    df = ['df', util.iris(['sepallength', 'sepalwidth',
+                           'petallength', 'petalwidth'], slice_size)]
+
+    arguments = {
+        'parameters': {},
+        'named_inputs': {
+            'input data': df[0],
+        },
+        'named_outputs': {
+            'split 1': 'split_1_task_1',
+            'split 2': 'split_2_task_1'
+        }
+    }
+    instance = SplitOperation(**arguments)
+    result = util.execute(instance.generate_code(),
+                          {'df': df[1]})
+
+    assert len(result['split_1_task_1']) and len(result['split_2_task_1']) != slice_size
+
+
+def test_split_fail_by_seed():
+    slice_size = 10
+    df = ['df', util.iris(['sepallength', 'sepalwidth',
+                           'petallength', 'petalwidth'], slice_size)]
+
+    arguments = {
+        'parameters': {'seed': 4294967296  # Seeds higher or equal to 4294967296, and lower than 0 break the code
```

Comment:
Yes, the GUI allows these out of range values. If you're comfortable, you may change the SplitOperation code to check the valid range.